Mini/MiniCooperSE.json

```
@@ -0,0 +1,71 @@
+{
+  "init_commands": {
+    "command": [
+	"ATZ", 
+	"ATD", 
+	"ATE0", 
+	"ATS0", 
+	"ATAL",
+	"ATPBE101",
+	"ATSPB",
+	"ATBI",
+	"ATSH6F1",
+	"ATAT0",
+	"ATSTFF",
+	"ATCRA607",
+        "ATCEA07",
+        "ATFCSH6F1",
+        "ATFCSD07300800",
+        "ATFCSM1"
+	]
+  },
+  "data_commands": {
+    "command": [
+      "22DDBC",
+      "22DD68",
+      "22DD69",
+      "226335"
+    ]
+  },
+  "obd_protocol": "B",
+  "soc": {
+    "equation": "(G*256+H)*0.1",
```

```suggestion
    "equation": "INT16(G:H)*0.1",
```

We offer this function which is a bit easier to read, but both variants work

---

A:B is correct here, so you can exchange every G->A, H->B, ...